core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1417,27 +1551,31 @@ class SparkContext(config: SparkConf) extends Logging {
 
   /**
    * Add a file to be downloaded with this Spark job on every node.
-   * The `path` passed can be either a local file, a file in HDFS (or other Hadoop-supported
-   * filesystems), or an HTTP, HTTPS or FTP URI.  To access the file in Spark jobs,
-   * use `SparkFiles.get(fileName)` to find its download location.
+   *
+   * @param path can be either a local file, a file in HDFS (or other Hadoop-supported
+   *             filesystems), or an HTTP, HTTPS or FTP URI.  To access the file in Spark jobs,
```

Comment:
nit: we could make this single spaced here` URI.  To` and also same instances, at least for the lines this PR changes.